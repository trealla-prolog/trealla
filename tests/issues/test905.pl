:- initialization(main).

main :-
	read_from_chars("%",T),
	write(T), nl.
