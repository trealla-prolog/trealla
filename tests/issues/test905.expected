   Warning: Initialization goal exception: error(existence_error(procedure,read_from_chars/2),read_from_chars/2).
